docs/source/make_a_release.rst

```diff
@@ -161,43 +161,71 @@ PyPI: Step-by-step
 Conda: Step-by-step
 -------------------
 
+.. warning::
+
+   Publishing on conda requires you to have corresponding package version uploaded on
+   pypi. So you have to do the pypi release before you do the conda release.
+
 In order to release a new version on conda-forge, follow the steps below:
 
-1. Clone feedstock
+1. Choose the next release version number (that matches with the pypi version that you already published)
+
+  .. code::
+
+    $ release=X.Y.Z
+
+2. Fork pynwb-feedstock
+
+ First step is to fork `pynwb-feedstock <https://github.com/conda-forge/pynwb-feedstock>`_ repository.
+ This is the recommended `best practice <https://conda-forge.org/docs/conda-forge_gotchas.html#using-a-fork-vs-a-branch-when-updating-a-recipe>`_  by conda.
+
+
+3. Clone forked feedstock
+
+   Fill the YOURGITHUBUSER part.
 
    .. code::
 
-      $ cd /tmp && \
-        git clone git@github.com:conda-forge/pynwb-feedstock.git
+      $ cd /tmp && git clone https://github.com/YOURGITHUBUSER/pynwb-feedstock.git
+
+
+4. Download corresponding source for the release version
+
+  .. warning::
+     You have to do the github and pypi release for this step to succeed.
```

Comment:
I suggest to remove this comment, there is already a warning at the top

Or if we keep it, the wording would have to be updated to be more accurate